tests/snapshots/test_cli__test-project-suggest.snap

```diff
@@ -1,22 +1,23 @@
 ---
 source: tests/test-cli.rs
-assertion_line: 158
 expression: stdout
 ---
-recommended audits:
-    atty:0.2.14         for safe-to-deploy  (508 lines)     (used by clap) 
-    hermit-abi:0.1.19   for safe-to-run     (938 lines)     (used by atty) 
-    termcolor:1.1.3     for safe-to-deploy  (2585 lines)    (used by clap) 
-    os_str_bytes:6.0.0  for safe-to-deploy  (2918 lines)    (used by clap) 
-    textwrap:0.15.0     for safe-to-deploy  (6165 lines)    (used by clap) 
-    indexmap:1.8.1      for safe-to-deploy  (9452 lines)    (used by clap, h2) 
-    reqwest:0.11.10     for safe-to-deploy  (21663 lines)   (used by test-project) 
-    serde_json:1.0.79   for safe-to-deploy  (22855 lines)   (used by reqwest, test-project) 
-    tokio:1.17.0        for safe-to-deploy  (91278 lines)   (used by h2, hyper, and 5 others) 
-    libc:0.2.123        for safe-to-run     (94067 lines)   (used by atty, core-foundation, and 10 others) 
-    winapi:0.3.9        for safe-to-run     (181329 lines)  (used by atty, mio, and 9 others) 
+recommended audits for safe-to-deploy:
+    cargo vet inspect atty 0.2.14         (used by clap)                              (508 lines)
+    cargo vet inspect termcolor 1.1.3     (used by clap)                              (2585 lines)
+    cargo vet inspect os_str_bytes 6.0.0  (used by clap)                              (2918 lines)
+    cargo vet inspect textwrap 0.15.0     (used by clap)                              (6165 lines)
+    cargo vet inspect indexmap 1.8.1      (used by h2, clap)                          (9452 lines)
+    cargo vet inspect reqwest 0.11.10     (used by test-project)                      (21663 lines)
+    cargo vet inspect serde_json 1.0.79   (used by reqwest, test-project)             (22855 lines)
+    cargo vet inspect tokio 1.17.0        (used by h2, hyper, reqwest, and 4 others)  (91278 lines)
```

Comment:
Hmm so one problem with this new ordering is that the column alignment now forces the high-variance thing to the middle instead of the end. This basically forces every line to have "worst-case width" and means if a terminal is a bit too narrow then ~every line will wrap instead of only the longest.

Reply:
I'm willing to live with this for now and see how many people complain.